tobler/util/util.py

```diff
@@ -35,8 +35,10 @@ def circumradius(resolution):
             "You can install it with `conda install h3-py` or "
             "`pip install h3`"
         )
-
-    return h3.edge_length(resolution, "m")
+    h3ver = int(h3.__version__[0])
+    if h3ver < 4:
```

Comment:
```suggestion
    if Version(h3.__version__) < Version("4.0"):
```
For the consistency (and robustness) with the same check below.